src/components/common/autoComplete/inputWithAutoComplete.tsx

```
@@ -0,0 +1,38 @@
+
+function InputWithAutoComplete(props: CustomInputProps) {
+  const { selectedOption, setIsBlur } = useAutoComplete();
+  const inputRef = useRef<null | HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (inputRef.current) inputRef.current.value = selectedOption;
+  }, [selectedOption]);
+
+  const handleBlur = (e: React.FocusEvent) => {
+    const relatedTarget = e.relatedTarget as HTMLElement;
+    if (relatedTarget && relatedTarget.closest('.auto-complete')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)